tests/wpt/servowpt.py

```diff
@@ -154,10 +154,23 @@ def run_tests(**kwargs):
 
     # Filter intermittents if that was specified on the command-line.
     if handler.unexpected_results and filter_intermittents_output:
-        all_filtered = filter_intermittents(
-            handler.unexpected_results,
-            filter_intermittents_output
-        )
+        unexpected_results = list(handler.unexpected_results)
```

Comment:
suggest commenting

```python
        # Start with the unexpected results from the first run. Some results may
        # be marked as flaky after rerunning, but the list won’t change.
```

Reply:
I've expanded your proposed comment a bit:

```
+        # Copy the list of unexpected results from the first run, so that we
+        # can access them after the tests are rerun (which changes
+        # `handler.unexpected_results`). After rerunning some tests will be
+        # marked as flaky but otherwise the contents of this original list
+        # won't change.
```